common/static/common/js/third_party_auth/grant_data_sharing_permissions.js

```diff
@@ -0,0 +1,29 @@
+(function(define) {
+    'use strict';
+
+    define(['jquery',
+            'underscore',
+            'gettext'],
+        function($, _, gettext) {
+            var REQUIRED_DATA_SHARING_STATES = {
+                required: {
+                    consent: "required",
+                    message: "Are you Sure ?\nIf you do not agree to share your data, you will not get any discounts from your organization."
+                },
+                enforced: {
+                    consent: "enforced",
+                    message: "Are you Sure ?\nIf you do not agree to share your data, you will have to use another account to access edx."
```

Comment:
need to mark these strings for I18N.  Also, we should reference settings.PLATFORM_NAME here and not 'edx'